test/e2e/spec/lightmode_spec.rb

```diff
@@ -0,0 +1,19 @@
+# coding: utf-8
+RSpec.describe "Light-mode E2E tests", :light do
+
+  describe "Syncing" do
+    it "I can restore empty wallet" do
+      wid = create_shelley_wallet
+      wait_for_shelley_wallet_to_sync(wid)
+      wallet = SHELLEY.wallets.get(wid)
+      expect(wallet['balance']['total']['quantity']).to eq 0
+    end
+
+    it "I can restore wallet with funds and I see ADA balance" do
+      wid = create_fixture_wallet(:shelley)
+      wait_for_shelley_wallet_to_sync(wid)
```

Comment:
This fixture wallet has ~18k transactions and growing. It seems that syncing it in light mode takes actually more time than syncing in full-mode (not sure if it is going to be feasible test then, need to test more)

Reply:
I can't find it right now, but when designing light mode, I did find out that it's fair to consider a person that does 200 transactions per year a day trader. 🤓 Light-mode won't be useful for people who do more transactions than this.

Reply:
Ok, I've added small wallet (only 1tx) for recurring e2e test for now.